### PR TITLE
feat:added svg in config.yml

### PR DESCRIPTION
**Pull request by Ghat0tkach**
**Description**

This PR fixes #492 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ghat0tkach** [comment]
@thisiskarangit please have a look .

File: .github/config.yml [hunk 1]
```diff
@@ -15,9 +15,13 @@ newPRWelcomeComment: >
...
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to the Meshery! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery-linkerd/master/.github/welcome/Layer5-celebration.svg)
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/meshery-celebration-light.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/meshery-celebration-dark.svg">
+  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
```
Comment (thisiskaransgit) [inline .github/config.yml:23]: Double check the path for the directory, we're in `meshery-linkerd`

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

File: .github/config.yml (hunk 1 above)
Comment (Ghat0tkach) [reply .github/config.yml]: thank you , I forgot bout these . Fixed it now

File: .github/config.yml [hunk 2]
```diff
@@ -15,9 +15,13 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to the Meshery! :tada:
```
Comment (Aisuko) [inline .github/config.yml:18]:
```suggestion
  Thanks for your contribution to the Meshery community! :tada:
```

**Review by Aisuko — CHANGES_REQUESTED** [review]

File: .github/config.yml (hunk 2 above)
Comment (Ghat0tkach) [reply .github/config.yml]: done

**Review by Aisuko — APPROVED** [review]
LGTM

**Pull request merged**